### PR TITLE
Add scripts and current cypher queries

### PR DESCRIPTION
**Pull request by robchamberspfc**
### What

Describe what you have changed and why.

### How to review

Describe the steps required to test the changes.

### Who can review

Describe who worked on the changes, so that other people can review.

### CODE REVIEW

File: scripts/codelists.json [hunk 1]
```diff
@@ -33,3 +33,4 @@
 {"_id":"opss-scheme-membership-sizeband","name":"schememembershipsizeband","links":{"self":{"id":"opss-scheme-membership-sizeband","href":"http://localhost:22400/code-lists/opss-scheme-membership-sizeband"},"codes":{"href":"http://localhost:22400/code-lists/opss-scheme-membership-sizeband/codes"}}}
 {"_id":"opss-status","name":"status","links":{"self":{"id":"opss-status","href":"http://localhost:22400/code-lists/opss-status"},"codes":{"href":"http://localhost:22400/code-lists/opss-status/codes"}}}
 {"_id":"wellbeing-geography","name":"geography","links":{"self":{"id":"wellbeing-geography","href":"http://localhost:22400/code-lists/wellbeing-geography"},"codes":{"href":"http://localhost:22400/code-lists/wellbeing-geography/codes"}}}
+{"_id":"mmm-yy","Name":"time","Links":{"self":{"id":"mmm-yy","href":"http://localhost:22400/code-lists/mmm-yy"},"codes":{"href":"http://localhost:22400/code-lists/mmm-yy/codes"}}}
```
Comment (eldeal) [inline scripts/codelists.json]: the fields Name and Links need to be lower cased

File: scripts/mongo-to-neo-migration/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Calculates dataset/edition and version adds these as labels to the instances in Neo4j
+
+dataset: "string"
+edition: "string"
+version: integer
+
+Also add published flag
+
+published_flag : true
```
Comment (eldeal) [inline scripts/mongo-to-neo-migration/README.md]: Your actual code uses `is_published` in line with the Dataset API, probably worth the documentation matching

File: scripts/mongo-to-neo-migration/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+### codelists.html
+
+Calulates the codelists and codes needed and writes the script tho import these and add the relationships between them.
```
Comment (eldeal) [inline scripts/mongo-to-neo-migration/README.md]: Typo in calculates and `script to import`

File: scripts/codelists.json (hunk 1 above)
Comment (robchamberspfc) [reply scripts/codelists.json]: Fixed

File: scripts/mongo-to-neo-migration/README.md (hunk 3 above)
Comment (robchamberspfc) [reply scripts/mongo-to-neo-migration/README.md]: Fixed

File: scripts/mongo-to-neo-migration/README.md (hunk 2 above)
Comment (robchamberspfc) [reply scripts/mongo-to-neo-migration/README.md]: Fixed

**Pull request merged**